build.ps1

```diff
@@ -107,7 +107,7 @@ $MODULES_PACKAGES_CONFIG = Join-Path $MODULES_DIR "packages.config"
 # INSTALL .NET CORE CLI
 ###########################################################################
 
-$DotNetChannel = "Current";
+$DotNetChannel = "2.1";
```

Comment:
What's the difference between DotNetChannel and DotNetVersion?  What is the alternative, if any, to using "Latest" for the latter?

Reply:
From: https://dot.net/v1/dotnet-install.ps1 (which is the script we're downloading and calling)
TLDR: The channel specifies the 2 part version number OR latest/ most current LTS.
The Version specifies a specific build version. The alternative to using the latest would be pinning to a specific version of .NET Core 2.1. I haven't researched this extensively given the time constraints, but latest seems that gives us the security patches for .NET Core 2.1 (see https://dotnet.microsoft.com/download/dotnet-core/2.1 as an example of what "latest" entails for 2.1)

> .PARAMETER Channel
    Default: LTS
    Download from the Channel specified. Possible values:
    - Current - most current release
    - LTS - most current supported release
    - 2-part version in a format A.B - represents a specific release
          examples: 2.0, 1.0
    - Branch name
          examples: release/2.0.0, Master
    Note: The version parameter overrides the channel parameter.
.PARAMETER Version
    Default: latest
    Represents a build version on specific channel. Possible values:
    - latest - most latest build on specific channel
    - coherent - most latest coherent build on specific channel
          coherent applies only to SDK downloads
    - 3-part version in a format A.B.C - represents specific version of build
          examples: 2.0.0-preview2-006120, 1.1.0